src/Compilers/Core/CodeAnalysisTest/EmbeddedTextTests.cs

```diff
@@ -171,24 +171,24 @@ public void FromBytes_LargeSpan()
         {
             var bytes = Encoding.Unicode.GetBytes(LargeSource);
             var paddedBytes = new byte[] { 0 }.Concat(bytes).Concat(new byte[] { 0 }).ToArray();
-            var checksum = SourceText.CalculateChecksum(bytes, 0, bytes.Length, SourceHashAlgorithm.Sha256);
-            var text = EmbeddedText.FromBytes("pathToLarge", new ArraySegment<byte>(paddedBytes, 1, bytes.Length), SourceHashAlgorithm.Sha256);
+            var checksum = SourceText.CalculateChecksum(bytes, 0, bytes.Length, SourceHashAlgorithms.Default);
+            var text = EmbeddedText.FromBytes("pathToLarge", new ArraySegment<byte>(paddedBytes, 1, bytes.Length), SourceHashAlgorithms.Default);
 
             Assert.Equal("pathToLarge", text.FilePath);
             AssertEx.Equal(checksum, text.Checksum);
-            Assert.Equal(SourceHashAlgorithm.Sha256, text.ChecksumAlgorithm);
+            Assert.Equal(SourceHashAlgorithms.Default, text.ChecksumAlgorithm);
             AssertEx.Equal(BitConverter.GetBytes(bytes.Length), text.Blob.Take(4));
             AssertEx.Equal(bytes, Decompress(text.Blob.Skip(4)));
         }
 
         [Fact]
         public void FromSource_Large()
         {
-            var source = SourceText.From(LargeSource, Encoding.Unicode, SourceHashAlgorithm.Sha256);
+            var source = SourceText.From(LargeSource, Encoding.Unicode, SourceHashAlgorithms.Default);
             var text = EmbeddedText.FromSource("pathToLarge", source);
 
             Assert.Equal("pathToLarge", text.FilePath);
-            Assert.Equal(SourceHashAlgorithm.Sha256, text.ChecksumAlgorithm);
+            Assert.Equal(SourceHashAlgorithms.Default, text.ChecksumAlgorithm);
```

Comment:
So we don't convert "default" to something else like an EffectiveSourceHashAlgorithm until somewhere much later?